lib/scss_lint/linter/merge_equal_selector.rb

```diff
@@ -0,0 +1,60 @@
+module SCSSLint
+  class Linter::MergeEqualSelector < Linter
+    include LinterRegistry
+
+    def check_node(node)
+      visits = Set.new
+      node.children.each do |child_node|
+
+        equal_node = find_equal_node child_node, node.children
+        next unless equal_node
+        next if visits.include? [child_node, equal_node] or
```

Comment:
The verbal `or`/`and` boolean operators aren't used anywhere else in this repository, and we'd like to keep it that way due to the way their precedence is handled. Please use `||`/`&&` instead.